HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.18 / 2021-07-20
```

Comment:
should this be updated to a more recent date since it's 9 days behind now?

Reply:
Sure, updated, thanks